transcom/mymove
# B 20943 INT

## [B-20943](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A997022)
## [B-20944](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A997025)
## [B-20945](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A997028)

## Summary

Fixes improper use of the SIT estimated entry and departure dates  instead of actual dates on payment packets. Also fixed issue where days in storage was calculated using estimated instead of actual dates for SIT on payment packets.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Milmove application
2. Login as a customer
3. Create a PPM, ensure that it has storage planned
4.  Submit the PPM
5. Login as a Service Counselor
6. Submit move details for your created move
7. Login as the Customer you create the PPM for
8. Upload documents and enter Storage Expense details for the PPM (pick different actual dates that estimated dates entered previously for ease of testing)
9. Login as a Service Counselor
10. Review the documents and approve the storage 
11. Login as the Customer
12. Download the Payment Packet
13. Ensure the Actual dates listed on the Storage Expense object are used on the packet, and not the estimated dates
14. Login as a Service Counselor
15. Locate the PPM you created
16. Download the Payment Packet
17. Ensure the Actual dates listed on the Storage Expense object are used on the packet, and not the estimated dates

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
